crates/codegen/runtime/cargo/src/runtime/bindings/mod.rs

```diff
@@ -1,16 +1,26 @@
 #[path = "generated/binding_rules.rs"]
 mod binding_rules;
 
-use metaslang_bindings;
+use std::sync::Arc;
```

Comment:
I don't think that decision is documented. I could start by adding documentation to the `jump_to_definition` method. Otherwise, I could write some in the [Bindings API Notion page](https://www.notion.so/nomicfoundation/Binding-API-d94be149123343c5a1e184a70f1be98b). WDYT?

> `import {Bar as LocalBar} from "bar.sol";`

`LocalBar` is a definition here, and `Bar` is the reference. If there is no alias, then the same identifier is both a reference and a definition.

> `type Price is uint128;`

The rules for UDVT are not fully implemented yet, but in this case `Price` would be strictly a definition. And since this creates a new type in Solidity, I expect all references to `Price` would point to this definition (ie. not bypass it to the underlying `uint128`).

Reply:
> I could start by adding documentation to the jump_to_definition method

If it won't derail your current progress, then adding documentation to the API sounds great.
If not, maybe we can add them to the binding rules file? it will also help us if we decide to change/update the rules in the future.

Reply:
Sure, I can add it.